dbt/README.md

```diff
@@ -48,14 +48,24 @@ Build the models to create views in our Athena warehouse:
 dbt run
 ```
 
+By default, all `dbt` commands will run against the `dev` environment, which
+namespaces the resources it creates by prefixing target database names with
+your Unix `$USER` name (e.g. `jecochr-default` for the `default` database when
```

Comment:
How does this work when the name of a branch includes an unexpected character (i.e. the `/` in `jeancochrane/28-data-catalog-add-production-profile-to-the-dbt-configuration`)?

Reply:
Good question! I think we can replace special characters (and uppercase letters, which are [not allowed in `dbt-athena`](https://dbt-athena.github.io/docs/known-issues)) using Jinja template filters. I'll take a stab at that now.

Reply:
@dfsnow This spiraled a little bit, but I was able to make it work in https://github.com/ccao-data/data-architecture/pull/44/commits/181e516d577c82add3b37a50c5e5e64c7a03d6d6! It required a new custom macro, but that led to me realize that we were starting to build a lot of logic into a collection of untested macros, so I took a minute to design a quick unit testing framework for our macros. Curious what you think!

Reply:
This is great! Nice work and definitely needed since this is getting more complex. I'm a dumb dumb and commented on a commit instead of the PR. Linking here: https://github.com/ccao-data/data-architecture/commit/181e516d577c82add3b37a50c5e5e64c7a03d6d6#r123228596